### ISSUE TITLE
Check to confirm that proposal links still work

### ISSUE THREAD

**Issue by prondubuisi** [prondubuisi/accepted-gsoc-proposals#21]
Need contributors who would check if proposal links are still active, this will insure that information here is still valid, please do bring to my attention any broken links.

Thanks, looking forward to your contribution.

**Comment by lumenCodes** [comment]
Please where can I see the link?

**Comment by prondubuisi** [comment]
Hello @lumenCodes you can navigate through the different project-proposal folders and click the links to find out if the are broken, or if the don't directly  point to projects and proposals. An example is the Jenkins proposals for 2019 https://github.com/prondubuisi/accepted-gsoc-proposals/blob/master/Jenkins/2019-proposals.md , the proposal links don't point to the proposals directly, but rather point to another document. You can then go ahead and create an issue pointing this out. If you want to fix the issue I am here to help too!

**Comment by lumenCodes** [comment]
Yes, I want to fix it.

**Comment by OyinOlamide** [comment]
Hello @prondubuisi Is this issue still open? I'd like to take it up.

**Comment by prondubuisi** [comment]
Yes @OyinOlamide , all issues on the Project are open. I look forward to your PR
